src/MizarData/MizarData.cpp

```diff
@@ -7,44 +7,67 @@
 #include <absl/container/flat_hash_set.h>
 
 #include <QStringLiteral>
+#include <filesystem>
 #include <iterator>
 #include <memory>
+#include <string>
 
 #include "ClientData/CallstackData.h"
 #include "ClientData/CallstackEvent.h"
+#include "ClientData/LinuxAddressInfo.h"
 #include "ClientData/ModuleAndFunctionLookup.h"
 #include "ClientData/ModuleData.h"
 #include "ClientData/ModuleManager.h"
+#include "ClientData/ProcessData.h"
 #include "ClientData/ScopeId.h"
 #include "ClientData/ScopeInfo.h"
 #include "ClientData/ThreadTrackDataProvider.h"
 #include "ClientSymbols/QSettingsBasedStorageManager.h"
 #include "GrpcProtos/symbol.pb.h"
 #include "MizarBase/AbsoluteAddress.h"
+#include "OrbitBase/Logging.h"
 #include "OrbitBase/Result.h"
 
 using ::orbit_mizar_base::AbsoluteAddress;
 using ::orbit_mizar_base::ForEachFrame;
 
 namespace orbit_mizar_data {
-[[nodiscard]] absl::flat_hash_map<AbsoluteAddress, std::string> MizarData::AllAddressToName()
-    const {
-  absl::flat_hash_map<AbsoluteAddress, std::string> result;
+
+absl::flat_hash_map<AbsoluteAddress, FunctionSymbol> MizarData::AllAddressToFunctionSymbol() const {
+  absl::flat_hash_map<AbsoluteAddress, FunctionSymbol> result;
 
   GetCaptureData().GetCallstackData().ForEachUniqueCallstack(
       [&result, this](const uint64_t /*callstack_id*/,
                       const orbit_client_data::CallstackInfo& info) {
         ForEachFrame(info.frames(), [this, &result](const AbsoluteAddress address) {
           std::optional<std::string> name = this->GetFunctionNameFromAddress(address);
           if (name.has_value()) {
-            result.try_emplace(address, std::move(name.value()));
+            std::string module_file_name = GetModuleFileName(address);
+            result.try_emplace(
+                address, FunctionSymbol{std::move(name.value()), std::move(module_file_name)});
           }
         });
       });
 
   return result;
 }
 
+[[nodiscard]] static std::string GetFilename(const std::string& path) {
+  return std::filesystem::path(path)
+      .filename()
+      .replace_extension()  // remove extention, so `app.exe` on Windows would match `app` on Linux
+      .string();
+}
+
+std::string MizarData::GetModuleFileName(AbsoluteAddress address) const {
```

Comment:
I would stay consistent with either `FileName` or `Filename` (I prefer the latter). Again, you could also add a `WithoutExtension` suffix here. When thinking about hebeckers talk the other day, I would prefer either having the suffix, or adding a comment in the header at the function. When I would try to reuse this function and only read the header file, I currently would assume I get a filename with extension.